src/Request/AbstractRequest.php

```diff
@@ -77,6 +98,11 @@ public function getParameters()
             $parameters['NUMAPPEL'] = $this->getCallNumber();
         }
 
+        // Direct Plus requests special case.
+        if (null !== $this->getSubscriberRef()) {
+            $parameters['REFABONNE'] = $this->getSubscriberRef();
```

Comment:
Use constants. Someinterface::CUSTOMER_REFERENCE = 'REFABONNE'; That way you don't have `REFABONNE` all over the place, just in one interface.